From 93812e7f214393dd000da75621951192320c57f5 Mon Sep 17 00:00:00 2001
From: Abinaya Kalaiselvan <akalaise@codeaurora.org>
Date: Fri, 18 Dec 2020 14:52:10 +0530
Subject: [PATCH] ath11k: Use latest mhi api

mhi_rddm_download_img function name was changed in mhi
core driver to mhi_rddm_download_image, Use this updated api.

Signed-off-by: Abinaya Kalaiselvan <akalaise@codeaurora.org>
---
 drivers/net/wireless/ath/ath11k/coredump.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath11k/coredump.c b/drivers/net/wireless/ath/ath11k/coredump.c
index 02a1347..ffe6df9 100644
--- a/drivers/net/wireless/ath/ath11k/coredump.c
+++ b/drivers/net/wireless/ath/ath11k/coredump.c
@@ -156,7 +156,7 @@ void ath11k_coredump_download_rddm(struct ath11k_base *ab)
 	struct ath11k_dump_segment *segment, *seg_info;
 	int i, rem_seg_cnt = 0, len, num_seg, seg_sz, qdss_seg_cnt = 1;
 
-	mhi_download_rddm_img(mhi_ctrl, false);
+	mhi_download_rddm_image(mhi_ctrl, false);
 
 	rddm_img = mhi_ctrl->rddm_image;
 	fw_img = mhi_ctrl->fbc_image;
-- 
2.7.4

